astro/src/content/docs/get-started/core-concepts/collected-metrics.mdx

```diff
@@ -0,0 +1,535 @@
+---
+title: Collected Metrics
+description: An overview of how, when, and why FusionAuth collects deployment usage data.
+section: get started
+subcategory: download and install
+---
+
+One of our goals is constant improvement. In order to get there, we collect some usage data from FusionAuth deployments. This data helps us understand how you're using FusionAuth so that we can give you better features, tightened security, and tailored experiences.
+
+Currently FusionAuth sends a payload to the metrics endpoint for all instances.
+
+Collected metrics of this payload include:
```

Comment:
Did you pull this out of the system requirements section? /docs/get-started/download-and-install/system-requirements#collected-metrics

If so, please update that section to point here.